internal/serve/serve.go

```diff
@@ -115,14 +89,6 @@ func Serve(cfg Configs) error {
 		Handler:    handler(deps),
 		OnStarting: func() {
 			log.Infof("🌐 Starting Wallet Backend server on port %d", cfg.Port)
-			go populatePools(ctx, deps.PoolPopulator)
-		},
-		OnStopping: func() {
-			log.Info("Stopping Wallet Backend server")
```

Comment:
I find it useful to log when the server is stopping. It may be useful when looking at server logs